contracts/Swap/SwapRouter.sol

```diff
@@ -14,22 +17,32 @@ import "./interfaces/IVBNB.sol";
  * @author 0xlucian
  */
 
-contract SwapRouter is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, RouterHelper, IPancakeSwapV2Router {
+contract SwapRouter is Ownable2StepUpgradeable, RouterHelper, IPancakeSwapV2Router {
+    using SafeERC20Upgradeable for IERC20Upgradeable;
+
+    address public comptrollerAddress;
```

Comment:
In the future, if we want to make this SwapRouter contract compatible with IL, I think we'll need to replace this address with the PoolRegistry. But we can do it later. Even we could move this swap features to a new repo...